### PR TITLE
editors/gummi: Reintroduce port and update to 0.8.1

### PR DESCRIPTION
**Pull request by yzgyyang**
- Update entry in MOVED

PR:		250913
Approved by:	lwhsu (mentor)

### CODE REVIEW

**Comment by yzgyyang** [comment]
One potential improvement: Update pkg-descr.

Other than that, this port looks good to my eyes - I will let it pass CI and do a quick ad-hoc test.

**Comment by lwhsu** [comment]
> Submitted by: Zoltan ALEXANDERSON BESSE [zab@zltech.eu](mailto:zab@zltech.eu)
Use `git commit --author`

**Comment by yzgyyang** [comment]
I was afraid that I cannot do that, but I actually can open a PR with someone else's commit ;)

File: editors/gummi/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created by: Hakisho Nukama <nukama@gmail.com>
+# $FreeBSD$
```
Comment (lwhsu) [inline editors/gummi/Makefile]: `$FreeBSD$` is deprecated.

**Review by lwhsu — CHANGES_REQUESTED** [review]

**Review by lwhsu — APPROVED** [review]

**Comment by yzgyyang** [comment]
gummi_screenshot

Runs fine on my 13.0-RELEASE desktop.

**Pull request merged**